packages/styled-components/src/utils/stylis.js

```diff
@@ -0,0 +1,100 @@
+import Stylis from 'stylis/stylis.min';
```

Comment:
*feel free to ignore, reading up on what's going on*

This feels like it should be taken care of by the bundler, but I'm probably just missing some backstory here. Is there a reason this references the minified version?

Reply:
https://github.com/thysultan/stylis.js/blob/4561e9bc830fccf1cb0e9e9838488b4d1d5cebf5/package.json#L3

Reply:
`stylis` commits the minified JS file, so I think we switched over at some point since it yielded better results, i.e. was more minified than what another minifier compressing styled-components + stylis might achieve. But that might've changed to be fair.